client/src/pages/ProfileCompleted.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import {
+  ProfileCompletedButtonsWrapper,
+  ProfileCompletedWrapper,
+  ProfileCompletedHeadingWrapper,
+  ProfileCompletedHeading,
+  ProfileCompletedHeader,
+  HeadingIcon,
+  StyledButton,
+} from "components/CompletedProfile/CompletedProfile";
+
+const ProfileCompleted = () => {
+  return (
+    <ProfileCompletedWrapper>
+      <ProfileCompletedHeader>
+        <ProfileCompletedHeadingWrapper>
+          <HeadingIcon />
+          <ProfileCompletedHeading>
+            Thank you for joining our community!
+          </ProfileCompletedHeading>
+        </ProfileCompletedHeadingWrapper>
+      </ProfileCompletedHeader>
+      <ProfileCompletedButtonsWrapper>
+        <StyledButton secondary={true}>View my Profile</StyledButton>
```

Comment:
Should we use `<Link >` or `<a />` here instead?

Reply:
yeah semantically a button has to be a button I guess... this has some consequences for accessibility and certain browser behavior etc.

Reply:
Yes

Reply:
I think we can use the history prop for routing in this case